tests/integration/test_cube_slice.py

```diff
@@ -0,0 +1,18 @@
+# from unittest import TestCase
+# import numpy as np
+
+# from cr.cube.cube_slice import CubeSlice
+
+# # from .fixtures import CAT_X_NUM_X_DATETIME
+# from .fixtures import ECON_GENDER_X_IDEOLOGY_WEIGHTED
+# # from .fixtures import UNIVARIATE_CATEGORICAL
+# # from .fixtures import PETS_ARRAY_X_PETS
+# # from .fixtures import PETS_X_PETS_ARRAY
+
+
+# class TestCubeSlice(TestCase):
+#     def test_cube_slice_(self):
+#         cs = CubeSlice(ECON_GENDER_X_IDEOLOGY_WEIGHTED)
+#         expected = CAT_X_CAT['value']
+#         actual = cube._table._cube
+#         self.assertEqual(actual, expected)gg
```

Comment:
If this test is covered elsewhere, I vote that we get rid of this file. However, I suspect that we do want to do some testing of the cube slicing machinery in general here, no?